curriculum/challenges/english/25-front-end-development/review-semantic-html/671a83934b61f64cefe87a61.md

```diff
@@ -22,26 +22,145 @@ Review the concepts below to prepare for the upcoming quiz.
 - **Navigation Section (`nav`) element**: represents a section with navigation links.
 - **Figure element**: used to contain illustrations and diagrams.
 - **Emphasis (`em`) element**: marks text that has stress emphasis.
+
+```html
+<p>
+    Never give up on <em>your</em> dreams.
+</p>
+```
+
 - **Idiomatic Text (`i`) element**: used for highlighting alternative voice or mood, idiomatic terms from another language, technical terms, and thoughts.
+
+```html
+<p>
+    There is a certain <i lang="fr">je ne sais quoi</i> in the air.
+</p>
+```
+
+The `lang` attribute inside the open `i` tag is used to specify the language of the content. In this case, the language would be French. The `i` element does not indicate if the text is important or not, it only shows that it's somehow different from the surrounding text.
+
 - **Strong Importance (`strong`) element**: marks text that has strong importance.
-- **Bring Attention To (`b`) element**: used to bring attention to text that is not important for the meaning of the content.
+
+```html
+<p>
+  <strong>Warning:</strong> This product may cause allergic reactions.
+</p>
+```
+
+- **Bring Attention To (`b`) element**: used to bring attention to text that is not important for the meaning of the content. It's commonly used to highlight keywords in summaries or product names in reviews.
+
+```html
+<p>
+    We tested several products, including the <b>SuperSound 3000</b> for audio quality, the <b>QuickCharge Pro</b> for fast charging, and the
+    <b>Ecoclean Vacuum</b> for cleaning. The first two performed well, but the
+    <b>Ecoclean Vacuum</b> did not meet expectations.
+</p>
+```
+
 - **Description List (`dl`) element**: used to represent a list of term-description groupings.
 - **Description Term (`dt`) element**: used to represent the term being defined.
 - **Description Details (`dd`) element**: used to represent the description of the term.
+
+```html
+<dl>
+    <dt>HTML</dt>
+    <dd>HyperText Markup Language</dd>
+    <dt>CSS</dt>
+    <dd>Cascading Style Sheets</dd>
+</dl>
+```
+
 - **Block Quotation (`blockquote`) element**: used to represent a section that is quoted from another source. This element has a `cite` attribute. The value of the `cite` attribute is the URL of the source.
```

Comment:
we can add a sentence here explaining the cit element before the example

Reply:
Just added a commit with indentation changes and cite element explanation. Let me know of any further corrections!